sdk/ai/ai-inference-rest/test/public/utils/recordedClient.ts

```diff
@@ -58,7 +47,5 @@ export async function createModelClient(
   recorder?: Recorder,
   options?: ClientOptions,
 ): Promise<ModelClient> {
-  const { endpoint, apiKey } = getEndpointAndAPIKeyFromResourceType(resourceType);
-  const credential = new AzureKeyCredential(apiKey);
-  return createClient(endpoint, credential, recorder?.configureClientOptions(options ?? {}));
+  return createClient(getEndpointFromResourceType(resourceType), createTestCredential(), recorder?.configureClientOptions(options ?? { credentials: { scopes: ["https://cognitiveservices.azure.com/.default"] } }));
```

Comment:
I wonder this scope is not being used by the client implementation?

Reply:
Default is https://ml.azure.com/.default, but entra ID is not expected to be supported for those endpoints until next month.